std/experimental/allocator/mallocator.d

```diff
@@ -399,24 +394,27 @@ version(CRuntime_DigitalMars)
 {
     void* m;
 
+    @nogc nothrow
+        size_t addr() { return cast(size_t) m; }
```

Comment:
I moved this in the unittest, even though the version block had no imports, as I can't imagine there's a good reason for adding `addr` to the global namespace as a public symbol. The loss of UFCS is mitigated by the fact that we always were calling this on `m`, so it actually gets shorter.

Reply:
The loss of UFCS might make this less intuitive. I know I had to double check what `addr` does. What is the gain in changing the function signature?

Do the function attributes and the function definition have different indents or is it just the web display?

Reply:
If I didn't change the function (and just made it a static), then it would become `addr(m)`. I changed it to m_addr to make it more obvious what it is.

The attributes were indented differently. This is what my editor did. But I have redone the function into a lambda, which looks quite nice.